data/json/construction.json

```
@@ -4909,7 +4923,6 @@
     "qualities": [ [ { "id": "SCREW", "level": 1 } ] ],
     "using": [ [ "soldering_standard", 20 ] ],
     "components": [ [ [ "kiln", 1 ] ], [ [ "cable", 5 ] ], [ [ "power_supply", 1 ] ] ],
```

Removed a note on an electric construction for no apparent reason?
```suggestion
    "components": [ [ [ "kiln", 1 ] ], [ [ "cable", 5 ] ], [ [ "power_supply", 1 ] ] ],
    "pre_note": "Will only work if constructed in/on a building that has an electric grid with a mounted battery.",
```